### PR TITLE
UI/UX improvements, Documentation improvements, Backend tweaks.

### PR DESCRIPTION
**Pull request by hiroferreira**
**UI/UX Improvements**
Use new UI to:
- Multi-project functionality (create, clean, delete projects) 
- Assets Management (manage assets straight on web-ui)
- Edit Feed through Google Sheets
- Configure and Download Google Ads CSV 
- Configure, generate and Upload video variations to YouTube.

**Backend Improvements**
- Python3 Migration
- minor tweaks to allow all UI/UX improvements to run.

**Documentation improvements**
Improved README.md and added a simple user manual.

### CODE REVIEW

**Comment by hiroferreira** [comment]
@googlebot I fixed it.

File: vogon.py [hunk 1]
```diff
@@ -31,22 +31,118 @@
...
-from adwords_video_csv import AwvCsv
 
-def generate_videos(config_file, youtube_upload, preview_line, flags):
+program_dir = os.path.abspath(os.path.dirname(__file__))
+stop_gen_threads = {}
+running_gen_threads = {}
+
+def stop_video_generation(project_dir):
+  global stop_gen_threads, running_gen_threads
+  print("cancelling video genration for %s"%project_dir)
```
Comment (cbreuel) [inline vogon.py]: genration -> generation

File: vogon.py [hunk 2]
```diff
@@ -31,22 +31,118 @@
...
+  stop_gen_threads[project_dir] = True
+  while True:
+    if project_dir not in running_gen_threads or len(running_gen_threads[project_dir]) == 0:
+      stop_gen_threads[project_dir] = False
+      print("cancelled video genration for %s"%project_dir)
```
Comment (cbreuel) [inline vogon.py]: genration -> generation

File: vogon.py [hunk 3]
```diff
@@ -82,93 +178,328 @@ def generate_videos(config_file, youtube_upload, preview_line, flags):
...
+                                     output_stream)
+      txt_input_files.append(i_file)
+
+    complex_filters.append(c_filter)
+    input_stream = output_stream
+  return complex_filters, txt_input_files
 
 def run_ffmpeg(img_args, filters, input_video, output_video, executable='ffmpeg'):
-    """Run the ffmpeg executable for the given input and filter spec.
+    """Run the ffmpeg executable for the given input an1d filter spec.
```
Comment (cbreuel) [inline vogon.py]: typo

**Review by cbreuel — COMMENTED** [review]
Notes:
* Do we have license to publish all files in base_project/assets, especially the font and video?
* There's a move static/vogon-ui.css → base_project/logs/.keep, is that correct?
* There are several other ".keep" files that I don't know what they're for, but looks like they already existed
* base_project/feed.csv is added as binary, should be text

File: vogon.py (hunk 3 above)
Comment (hiroferreira) [reply vogon.py]: Fixed on [next commit](https://github.com/google/vogon/pull/10/commits/1d33650283805c1ae7316c999b2964c3ef5ed8e8)

File: vogon.py (hunk 2 above)
Comment (hiroferreira) [reply vogon.py]: Fixed on [next commit](https://github.com/google/vogon/pull/10/commits/1d33650283805c1ae7316c999b2964c3ef5ed8e8)

File: vogon.py (hunk 1 above)
Comment (hiroferreira) [reply vogon.py]: Fixed on [next commit](https://github.com/google/vogon/pull/10/commits/1d33650283805c1ae7316c999b2964c3ef5ed8e8)

File: README.md [hunk 4]
```diff
@@ -1,95 +1,183 @@
...
+* Install ffmpeg and imagemagick
+* install GIT
+* Install Python3 and pip3
+* install Python libs with pip
+  * google-api-python-client
+  * oauth2client
+  * bottle
+  * http.client
+  * httplib2
+* Download Vogon codeAfter tht
```
[the remark this answers is not in the document]
Comment (hiroferreira) [reply README.md]: Fixed on [next commit](https://github.com/google/vogon/pull/10/commits/1d33650283805c1ae7316c999b2964c3ef5ed8e8)

File: .gitignore [hunk 5]
```diff
@@ -1,3 +1,8 @@
 *.pyc
 *-oauth2.json
 .DS_Store
+projects/
+tmp/
+log/
+credentials/*.json
+.*DS_Store
```
[the remark this answers is not in the document]
Comment (hiroferreira) [reply .gitignore]: Fixed on [next commit](https://github.com/google/vogon/pull/10/commits/1d33650283805c1ae7316c999b2964c3ef5ed8e8)

**Comment by hiroferreira** [comment]
> Notes:
> 
> * Do we have license to publish all files in base_project/assets, especially the font and video?
> * There's a move static/vogon-ui.css → base_project/logs/.keep, is that correct?
> * There are several other ".keep" files that I don't know what they're for, but looks like they already existed
> * base_project/feed.csv is added as binary, should be text

hey, just fixed all your notes on [next commit](https://github.com/google/vogon/pull/10/commits/1d33650283805c1ae7316c999b2964c3ef5ed8e8). Thank you for the time and concern. Could you PTAL?

**Review by cbreuel — APPROVED** [review]
LGTM, thanks!

**Pull request merged**